docs/documentation/securing_apps/topics/oidc/oauth21-support.adoc

```diff
@@ -0,0 +1,15 @@
+[[_oauth21-support]]
+=== OAuth 2.1 Support
+
+{project_name} makes it easier for administrators to make sure that their clients are compliant with these specifications:
+
+* https://datatracker.ietf.org/doc/html/draft-ietf-oauth-v2-1-10[The OAuth 2.1 Authorization Framework]
```

Comment:
```suggestion
* https://datatracker.ietf.org/doc/html/draft-ietf-oauth-v2-1-10[The OAuth 2.1 Authorization Framework - draft specification]
```

I would explicitly mention that OAuth 2.1 is still draft specification. Maybe we can also add some note like this at the end of your `OAuth 2.1 client profiles` section:
```
NOTE: OAuth 2.1 specification is still a draft and it may change in the future. Hence the {project_name} built-in OAuth 2.1 client profiles can change as well.
```

WDYT?

Reply:
Yes, I fixed the sentence as you suggested.